gateleen-routing/src/main/resources/gateleen_routing_schema_routing_rules

```diff
@@ -57,6 +57,11 @@
 					"type": "integer",
 					"default": 30
 				},
+				"keepAliveTimeout": {
+                    "description": "The keep alive timeout for HTTP/1.1 connections.",
```

Comment:
Suggest to mention unit in description.

Reply:
Missed this right, in the meantime we have merged. I'll try to remember this for my next PR.